src/main/java/core/basesyntax/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Driver;
+import core.basesyntax.service.CarService;
+import core.basesyntax.service.DriverService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final String PACKAGE_NAME = "core.basesyntax";
+    private static final Injector injector = Injector.getInstance(PACKAGE_NAME);
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/addDriverToCar.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/drivers/addDriverToCar.jsp").forward(req, resp);
```

Reply:
but we should keep such structure only in URL's, or in project tree too?

Reply:
its better to divide what affects only cars and what additionally affects drivers. just a suggestion

Reply:
or not divide, highlight

Reply:
done